src/decays/decay_functions.cpp

```diff
@@ -305,24 +305,15 @@ unsigned int number_of_active_flavours(softsusy::QedQcd const& qedqcd, double m)
 
 double sm_up_quark_masses(softsusy::QedQcd const& qedqcd, int n)
 {
-   switch(n) {
-      case 0: return qedqcd.displayMass(softsusy::mUp);
-      case 1: return qedqcd.displayMass(softsusy::mCharm);
-      default:
-         throw std::runtime_error("Unknown quark mass");
+   if (n == 2) {
+      throw std::runtime_error("sm_up_quark_masses: top quark mass cannot be used");
```

Comment:
Why excluded a top quark? We also need it.

Reply:
It was excluded before. :) If we need it, I'll modify this PR.

Reply:
Done in 3c89d4163

Reply:
Oh, sorry. I only vague remember why I did it like that :cry:

Reply:
Just be sure: We must include the top quark, right? I.e. the current status of this PR is correct, right? :)